src/content/en/fundamentals/performance/rail.md

```diff
@@ -108,8 +119,7 @@ Working in 50ms blocks allows the task to finish while still ensuring instant re
 
 ## Load: deliver content under 1000ms
 
-Load your site in under 1 second.
-If you don't, your user's attention wanders,
+Load your site in under 1 second. If you don't, user attention wanders,
```

Comment:
Note for a future set of edits: the RAIL guidelines we published to /web didn't factor in 3G and mobile devices as much as they could and so the 1s guideline here is a little nebulous (today we focus more on being interactive in <5s and hitting a Speed Index of < 4000ms on 3G.